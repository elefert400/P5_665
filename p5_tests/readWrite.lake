int main(){
    int x;
    bool b;
    x = 1;
    b = true;
    read x;
    write x;
    read b;
    write b;
}