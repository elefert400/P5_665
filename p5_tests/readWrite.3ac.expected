[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN main LOCALS]
x (local)
b (local)
[END main LOCALS]
enter main
x := 1
b := 1
READ x
WRITE x
READ b
WRITE b
lbl_0: EXIT